### ISSUE TITLE
rails g bootstrap:themed doesn't work on scaffold within subdirectory

### ISSUE THREAD

**Issue by artisr** [seyhunak/twitter-bootstrap-rails#323]
Lets say you generate the following scaffold and want to generate the bootstrap theme

```
rails g scaffold admin/post title:string description:text
rails g bootstrap:themed admin/posts -f
```

then you will get the following error when trying to view...

```
 uninitialized constant ActionView::CompiledTemplates::Post
```

Basically the view is being generated without regarding sub directory ..

lets say to fix index action, we need to change two lines in views/admin/posts/index.html.erb

```
<%- model_class = Post.new.class -%>
```

needs to read

```
<%- model_class = Admin::Post.new.class -%>
```

and 

```
<% @posts.each do |post| %>
```

needs to read

```
<% @admin_posts.each do |post| %>
```

I haven't looked at other actions..

So basically, it doesn't generate correct views..
